nx-X11/programs/Xserver/hw/nxagent/Clipboard.c

```diff
@@ -2394,16 +2394,7 @@ static void setSelectionOwnerOnXServer(Selection *pSelection)
   }
 
   #ifdef DEBUG
-  fprintf(stderr, "%s: lastSelectionOwner.client %s -> %s\n", __func__,
-              nxagentClientInfoString(lastSelectionOwner[index].client),
-                  nxagentClientInfoString(pSelection->client));
-  fprintf(stderr, "%s: lastSelectionOwner.window [0x%x] -> [0x%x]\n", __func__,
-              lastSelectionOwner[index].window, pSelection->window);
-  fprintf(stderr, "%s: lastSelectionOwner.windowPtr [%p] -> [%p] [0x%x] (serverWindow: [0x%lx])\n", __func__,
-              (void *)lastSelectionOwner[index].windowPtr, (void *)pSelection->pWin,
-                  pSelection->pWin ? nxagentWindow(pSelection->pWin) : 0, serverWindow);
-  fprintf(stderr, "%s: lastSelectionOwner.lastTimeChanged [%u]\n", __func__,
-              lastSelectionOwner[index].lastTimeChanged);
+  printSelectionStat(index);
```

Comment:
Shouldn't the part above be shipped in a separate commit?

Reply:
This doesn't feel related to the TimeStamp data type change...

Reply:
yes, it does relate to the TimeStamp data type change. I would have needed to change all those printfs where the lastTimeChanged was referred to. The easier way was to replace those printfs by a call to central subroutine that already printed everthing and was already uptodate regarding the TimeStamp change.